pkg/metrics_store/metrics_store.go

```diff
@@ -40,23 +40,23 @@ type MetricsStore struct {
 	// metric families, containing a slice of metrics. We need to keep metrics
 	// grouped by metric families in order to zip families with their help text in
 	// MetricsStore.WriteAll().
-	metrics map[types.UID][]string
+	metrics map[types.UID][][]byte
```

Comment:
Ping @LiliC: These patches make the metrics store cache byte slices instead of strings for performance reasons. You might need to adjust your logic in the operator framework to work with these patches.

Reply:
@mxinden Thanks for the heads up!